recipes-core/images/iot2050-image-example.bb

```diff
@@ -35,6 +35,7 @@ IMAGE_INSTALL += " \
     customizations-example \
     switchserialmode \
     iot2050-firmware-update \
+    iot2050-module-firmware-update \
```

Comment:
I don't get the commit message in line with with addition of the "module" update package to the image.